droidlet/interpreter/robot/default_behaviors.py

```diff
@@ -3,8 +3,47 @@
 """
 import logging
 from droidlet.interpreter.robot import tasks
+import os
+import random
+import numpy as np
 
+random.seed(2021) # fixing a random seed to fix default exploration goal
+first_exploration_done = False
+
+def get_distant_goal(x, y, t, l1_thresh=35):
+    # Get a distant goal for the slam exploration
+    # Pick a random quadrant, get 
+    while True:
+        xt = random.randint(-19, 19)
+        yt = random.randint(-19, 19)
+        d = np.linalg.norm(np.asarray([x,y]) - np.asarray([xt,yt]), ord=1)
+        if d > l1_thresh:
+            return (xt, yt, 0)
+
+def start_explore(agent, goal):
+    global first_exploration_done
```

Comment:
why not store this in memory, or in agent?  why a global?

Reply:
Agree, but wasn't sure of the best way to just have generic state (like this) in memory. Could you point me to a similar example? Or what would be a good way to do this?